src/adam_impact_study/conversions.py

```diff
@@ -56,85 +65,28 @@ def impactor_to_adam_orbit(impactor_df):
     return orbit
 
 
-def sorcha_output_to_df(sorcha_output_file):
+def sorcha_output_to_od_observations(sorcha_output_file):
     """
-    Generate a DataFrame from a Sorcha output file.
+    Convert Sorcha observations output files to Observations type.
 
     Parameters
     ----------
     sorcha_output_file : str
-        Path to the Sorcha output CSV file.
+        Path to the Sorcha output file.
 
     Returns
     -------
-    sorcha_output_df : pandas.DataFrame
-        DataFrame containing the Sorcha output data, sorted by Object ID
-        and observation time (fieldMJD_TAI).
-    """
-    sorcha_output_df = pd.read_csv(sorcha_output_file, float_precision="round_trip")
-    # sort by object id
-    sorcha_output_df = sorcha_output_df.sort_values(
-        by=["ObjID", "fieldMJD_TAI"], ignore_index=True
-    )
-    return sorcha_output_df
-
-
-def adam_orbit_from_sorcha_input(sorcha_orbits_file):
+    od_observations : qv.Table
+        Observations object continaining the Sorcha observations.
     """
-    Convert orbits in sorcha format to an ADAM Orbit object.
 
-    Parameters
-    ----------
-    sorcha_orbits_file : str
-        Path to the Sorcha orbits file.
-
-    Returns
-    -------
-    orbit : `~adam_core.orbits.orbits.Orbits`
-        ADAM Orbit object created from the Sorcha orbit data.
-    """
-    sorcha_orbits_df = pd.read_csv(sorcha_orbits_file, sep=" ")
-    keplerian_coords = KeplerianCoordinates.from_kwargs(
-        a=sorcha_orbits_df["a"],
-        e=sorcha_orbits_df["e"],
-        i=sorcha_orbits_df["inc"],
-        raan=sorcha_orbits_df["node"],
-        ap=sorcha_orbits_df["argPeri"],
-        M=sorcha_orbits_df["ma"],
-        time=Timestamp.from_mjd(sorcha_orbits_df["epochMJD_TDB"].values, scale="tdb"),
-        origin=Origin.from_kwargs(
-            code=np.full(len(sorcha_orbits_df), "SUN", dtype="object")
-        ),
-        frame="ecliptic",
+    sorcha_observations_df = pd.read_csv(
```

Comment:
Similarly, this and all the downstream column processing could be done directly in pyarrow (to numpy when necessary). I only mention it because I trust the pyarrow float parsing more than the pandas implementation.